scripts/configure-cluster/common/auth.sh

```diff
@@ -1,30 +1,10 @@
 #!/usr/bin/env bash
 
-set -e
-
-HTPASSWD_FILE="./htpass"
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+HTPASSWD_FILE="${SCRIPTPATH}/test-htpasswd"
 HTPASSWD_SECRET="htpasswd-secret"
 USERPASS="password@123"
```

Comment:
same here - is this needed?

Reply:
Ok as its used only in one location, removing for now